chart/templates/mayastor/agents/core/agent-core-and-ha-cluster-deployment.yaml

```diff
@@ -1,18 +1,18 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Release.Name }}-agent-core
+  name: {{ .Release.Name }}-agent-core-and-ha-cluster
```

Comment:
I'm not sure about this, I think in this case we should still let this be the core agent, otherwise as we bring features in we'd end up with a never ending name.
Also we have tests which depend on `app: agent-core` so I don't think we should change this.

Reply:
Okay.